syntax = "proto2";

package cal;

message CalHeader {
  required fixed32 msg_len = 1; 
};
